doc/source/user_guide/temperature.rst

```diff
@@ -16,16 +16,15 @@ Conversion is handled naturally through arithmetic operations:
 
     ureg = UnitRegistry()
 
-    K = Quantity(value=2, units=ureg.K)
-    K2 = Quantity(value=5, units=ureg.K)
-    delta_K = Quantity(value=7, units=ureg.delta_K)
-
-    K_ad_K2 = K + K2  # (9, K)
-    K2_sb_K = K2 - K  # (5, delta_K)
-    K_ad_delta_K = K + delta_K  # (7, K)
-    K2_sb_delta_K = K2 - delta_K  # (2, K)
-    delta_K_ad_delta_K = delta_K + delta_K  # (10, delta_K)
-    delta_K_sb_delta_K = delta_K - delta_K  # (0, delta_K)
+    K = Quantity(value=280, units=ureg.K)
```

Comment:
Please update the above text:

PyAnsys Units supports temperature units in kelvin (K), celsius (C), rankine (R)
and fahrenheit (F).

Conversion is handled naturally through arithmetic operations.

For each temperature unit there is a corresponding
temperature difference unit which is prefixed with ``delta_``.
In order to add two temperature-based quantities, at least one must be a 
temperature difference. If one temperature is substracted from another,
the result is a temperature difference.

Reply:
Done. Thank you.